cdk/domino_cdk/provisioners/efs.py

```diff
@@ -34,15 +34,14 @@ def provision_efs(self, stack_name: str, cfg: config.EFS, vpc: ec2.Vpc, security
             self.scope,
             "Efs",
             vpc=vpc,
-            # encrypted=True,
+            encrypted=True,
```

Comment:
What are the costs? Should it be an option? Why it was commented out?

Reply:
No cost associated, this is the default for existing EKS deploys. No sure why it was commented out tbh.

Reply:
I think it was to investigate this documentation comment that got cut off in the docs:

```
encrypted (Optional[bool]) – Defines if the data at rest in the file system is encrypted or not. Default: - If your application has the ‘
```

Looking at the code, it's this:

```
   * @default - If your application has the '@aws-cdk/aws-efs:defaultEncryptionAtRest' feature flag set, the default is true, otherwise, the default is false.
```

(so we do need it)

Also wondering, do we need to do anything with kms here?